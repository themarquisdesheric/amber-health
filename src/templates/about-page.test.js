import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import AboutPage from './about-page';

const data = {
  markdownRemark: {
    html: 'some markup'
  }
};

test('AboutPage', () => {
  const wrapper = shallow(<AboutPage data={data} />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
