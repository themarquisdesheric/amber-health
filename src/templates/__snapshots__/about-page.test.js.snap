// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage 1`] = `
<TemplateWrapper>
  <AboutPageTemplate
    content="some markup"
    contentComponent={[Function]}
  />
</TemplateWrapper>
`;
